manager/dashbar_handler.py

```diff
@@ -70,7 +70,7 @@ def handle_get(cls, **kwargs):
                                                 SystemPlatform.advisor_evaluated.is_null(False)) &
                                                (fn.COALESCE(SystemPlatform.host_type, 'null') != HostType.EDGE)))
                      .where(SystemVulnerabilities.rh_account_id == rh_account)
-                     .where(SystemVulnerabilities.mitigation_reason.is_null(True)))
+                     .where(SystemVulnerabilities.when_mitigated.is_null(True)))
```

Comment:
```suggestion
                     .where((SystemVulnerabilities.when_mitigated.is_null(True)) | ((InsightsRule.active == True) & (InsightsRule.rule_only == False))
```

Reply:
And you can remove the conditions in the ingishts rule join conditions (`active` and `rule_only`)